### PR TITLE
Fetch ambassadors from website API

### PR DESCRIPTION
**Pull request by MattIPv4**
cc https://github.com/alveusgg/alveusgg/pull/897 https://github.com/alveusgg/data/pull/88

### CODE REVIEW

File: src/components/AmbassadorCard.tsx [hunk 1]
```diff
@@ -259,7 +255,8 @@ export default forwardRef(function AmbassadorCard(
             <h3 className={headingClass}>Species Lifespan</h3>
             <p>
               Wild:{" "}
-              {"wild" in ambassador.lifespan ? (
+              {"wild" in ambassador.lifespan &&
+              ambassador.lifespan.wild !== undefined ? (
```
Comment (MattIPv4) [inline src/components/AmbassadorCard.tsx:259]: Is this change needed, this seems like a type issue?

File: src/components/AmbassadorCard.tsx [hunk 2]
```diff
@@ -272,7 +269,8 @@ export default forwardRef(function AmbassadorCard(
             </p>
             <p>
               Captivity:{" "}
-              {"captivity" in ambassador.lifespan ? (
+              {"captivity" in ambassador.lifespan &&
+              ambassador.lifespan.captivity !== undefined ? (
```
Comment (MattIPv4) [inline src/components/AmbassadorCard.tsx:273]: Is this change needed, this seems like a type issue?

**Review by pjeweb — APPROVED** [review]
